Recent PR builds failing when dependencies not found

**Do you want to request a *feature* or report a *bug*?**
A bug.

**What is the current behavior?**
The most recent PRs are getting failures in CI because `ReactDOM. Required module not found` and ` Cannot find module 'react/lib/ReactTestUtils'`.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. You can use this jsfiddle to get started: https://jsfiddle.net/stopachka/m6z0xn4r/.**
It seems like making any PR will trigger this error.

**What is the expected behavior?**
The tests should not fail for reasons unrelated to the PR.

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
This is in CI on Github.

The issue is only related to the AMD format. Does Draft use AMD somewhere?

>Cannot find module 'react/lib/ReactTestUtils'.

It looks like it is using incompatible versions of `react` and `react-dom`. They must be updated at the same time.

Can you check if bumping both to 15.4.0 fixes the issue

--- a/package.json
+++ b/package.json
@@ -35,8 +35,8 @@
     "object-assign": "^4.1.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || >=15.0.0-rc.1 <=15.3.2",
-    "react-dom": "^0.14.0 || >=15.0.0-rc.1 <=15.3.2"
+    "react": "^0.14.0 || ^15.0.0-rc",
+    "react-dom": "^0.14.0 || ^15.0.0-rc"
   },
   "devDependencies": {
     "babel-core": "^6.8.0",
@@ -64,8 +64,8 @@
     "gulp-uglify": "^1.2.0",
     "gulp-util": "^3.0.6",
     "jest": "^15.1.1",
-    "react": ">=15.0.0-rc.1 <=15.3.2",
-    "react-dom": ">=15.0.0-rc.1 <=15.3.2",
+    "react": "^15.0.0-rc.1",
+    "react-dom": "^15.0.0-rc.1",
     "run-sequence": "^1.1.2",
     "through2": "^2.0.1",
     "vinyl-buffer": "^1.0.0",

--- a/scripts/module-map.js
+++ b/scripts/module-map.js
@@ -14,8 +14,8 @@ module.exports = Object.assign(
     ReactDOM: 'react-dom',
     'object-assign': 'object-assign',
 
-    ReactTestUtils: 'react/lib/ReactTestUtils',
-    reactComponentExpect: 'react/lib/reactComponentExpect',
+    ReactTestUtils: 'react-dom/lib/ReactTestUtils',
+    reactComponentExpect: 'react-dom/lib/reactComponentExpect',
   },
   require('fbjs/module-map'),
   require('fbjs-scripts/third-party-module-map')

